apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisFiltersAddMoreButton.vue

```diff
@@ -0,0 +1,114 @@
+<script setup lang="ts">
+import { onClickOutside } from '@vueuse/core';
+import {
+    computed, defineEmits, reactive, ref, toRef, watch,
+} from 'vue';
+
+import {
+    PButton, PContextMenu, useContextMenuController,
+} from '@spaceone/design-system';
+import type { MenuItem } from '@spaceone/design-system/types/inputs/context-menu/type';
+
+import { useCostAnalysisPageStore } from '@/services/cost-explorer/store/cost-analysis-page-store';
+
+
+const emit = defineEmits<{(e: 'disable-filter', value: string): void;
+    (e: 'disable-all-filters'): void;
+}>();
+
+const costAnalysisPageStore = useCostAnalysisPageStore();
+const costAnalysisPageState = costAnalysisPageStore.$state;
+
+const state = reactive({
+    menuItems: computed<MenuItem[]>(() => costAnalysisPageStore.defaultGroupByItems),
+    selectedItems: [] as MenuItem[],
+    searchText: '',
+});
+
+const contextMenuRef = ref<any|null>(null);
+const targetRef = ref<HTMLElement | null>(null);
+const {
+    visibleMenu,
+    refinedMenu,
+    contextMenuStyle,
+    showContextMenu,
+    hideContextMenu,
+    initiateMenu,
+} = useContextMenuController({
+    targetRef,
+    contextMenuRef,
+    useReorderBySelection: true,
+    menu: toRef(state, 'menuItems'),
+    selected: toRef(state, 'selectedItems'),
+    searchText: toRef(state, 'searchText'),
+});
+onClickOutside(contextMenuRef, hideContextMenu);
+
+const handleClickAddMore = () => {
+    if (visibleMenu.value) {
+        hideContextMenu();
+    } else {
+        initiateMenu();
+        showContextMenu();
+    }
+};
```

Comment:
It seems like this code isn't working.
(I checked it in the preview.)

Reply:
I fixed it!